apis/v1alpha1/backendpolicy_types.go

```diff
@@ -167,6 +167,7 @@ type BackendTLSConfig struct {
 type BackendPolicyStatus struct {
 	// Conditions describe the current conditions of the BackendPolicy.
 	// +optional
+	// +kubebuilder:validation:MaxItems=8
```

Comment:
What is the forward/backward compatibility issues with being conservative here? Is this something that can be changed easily w/ a backward compatible change? I remember changing validation can be hard to get right.

Reply:
Validation is definitely difficult to get right, but I think it's generally easier to loosen it than tighten it. I know there's definitely precedent for loosening validation in k/k (including the Ingress API). The guidance I remember from Tim was to start with relatively strict validation and loosen it if/when we need to.